Neos.Flow/Classes/Http/Helper/ArgumentsHelper.php

```diff
@@ -43,12 +43,10 @@ public static function buildUnifiedArguments(array $getArguments, array $postArg
     /**
      * @param string $content
      * @return ContentStream
+     * @deprecated since Flow 5.2, use ContentStream::fromContents()
```

Comment:
That one had a short lifetime :smile:

Reply:
IMHO we could even remove it immediately.It's just in since the last release and the release notes stated that things are in motion atm and people shouldn't rely on stability on these APIs yet.

Reply:
Yeah, I remember that. Also doubt anyone has really jumped onto this API yet.

Reply:
I'd rather leave it in for now. It doesn't really harm and I'm pretty sure someone somewhere (for example we ourself in the core) already use it